DeAceroDLabs/marketplace-frontend
# [DMP-638] Lock unavailable properties

## What is done

[[DMP-638] Bloquear valores no disponibles](https://deacero.atlassian.net/browse/DMP-638)

## How is it done

1. Added a new function to validate that the unselected properties are available to select based on the selected ones.
2. Added a new color property to the button.
3. Modified the validation that indicated whether a product had been selected.
4. Modified the validation that indicated whether a product was available in the catalog.

## How to test

1. Select a product.
2. Click on a property value.
3. Check how the values start being blocked as they don't match with the selected properties' values
